libs/ledger-live-common/src/families/vechain/__snapshots__/bridge.integration.test.ts.snap

```diff
@@ -109,14 +109,14 @@ exports[`vechain currency bridge scanAccounts vechain seed 1 2`] = `
       "extra": {},
       "fee": "365180000000000000",
       "hash": "0x5fdd7191c4d476a8e86060d516366e87421f65667b9b3c14c33a740c04921b10",
-      "id": "js:2:vechain:0x0fe6688548f0C303932bB197B0A96034f1d74dba:vechain+vechain%2Fvip180%2Fvtho-0x5fdd7191c4d476a8e86060d516366e87421f65667b9b3c14c33a740c04921b10-IN",
+      "id": "js:2:vechain:0x0fe6688548f0C303932bB197B0A96034f1d74dba:vechain+vechain%2Fvip180%2Fvtho-0x5fdd7191c4d476a8e86060d516366e87421f65667b9b3c14c33a740c04921b10-OUT",
```

Comment:
Is it a regression ? it should stay as "IN"
https://explore.vechain.org/transactions/0x5fdd7191c4d476a8e86060d516366e87421f65667b9b3c14c33a740c04921b10#info